otter/integration/lib/mimic.py

```diff
@@ -71,4 +80,37 @@ def sequenced_behaviors(self, rcs, criteria, behaviors,
             }),
             headers=headers(str(rcs.token)),
             pool=self.pool
-        ).addCallback(check_success, [201]).addCallback(self.treq.content)
+        ).addCallback(check_success, [201]).addCallback(self.treq.json_content)
+
+        behavior_id = body['id']
+
+        if add_cleanup_to is not None:
+            add_cleanup_to.addCleanup(
+                self.delete_behavior, rcs, behavior_id, event_description,
+                [204])
+
+        returnValue(behavior_id)
+
+    def delete_behavior(self, rcs, behavior_id, event_description="creation",
+                        success_codes=None):
```

Comment:
You're right, it doesn't - I was following a pattern which doesn't really need to apply here unless we have some use for it.  Thanks :)